cdi-unit/src/main/java/org/jglue/cdiunit/internal/WeldTestUrlDeployment.java

```diff
@@ -93,7 +93,7 @@
 	private Set<URL> cdiClasspathEntries = new HashSet<URL>();
 	private final ServiceRegistry serviceRegistry = new SimpleServiceRegistry();
 
-	public WeldTestUrlDeployment(ResourceLoader resourceLoader, Bootstrap bootstrap, Class<?> testClass, Method testMethod) throws IOException {
+	public WeldTestUrlDeployment(ResourceLoader resourceLoader, Bootstrap bootstrap, TestConfiguration testConfiguration) throws IOException {
```

Comment:
Just as a matter of style, I suggest shortening the param name to testConfig, to keep the lines from getting too long.